ext/nnstreamer/tensor_decoder/meson.build

```diff
@@ -114,3 +114,31 @@ static_library('nnstreamer_decoder_image_segment',
   install: true,
   install_dir: nnstreamer_libdir
 )
+
+# flatbuffer
+if get_option('enable-flatbuffers')
+  run_command('/usr/bin/flatc', '--cpp', 'nnstreamer.fbs')
```

Comment:
1. Do not use hardcoded path of a binary. Use find_program or other portable methods.
2. Instead of run_command, use generator which is added to handle such cases (e.g., protoc)

Reply:
I could change it with your help. Thank you!!!